<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>880</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_comp_load">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Загрузить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_comp_save">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>сохранить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_comp_unite">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>объединение</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_comp_find">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>найти</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_comp_find_name">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_comp_find_result">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_comp_united">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>610</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_comp_second">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_comp_first">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_bag_unite">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>объединение</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_bag_united">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>640</y>
      <width>461</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_bag_find_result">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>100</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_bag_second">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>370</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_bag_load">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>60</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Загрузить</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_bag_find_name">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>60</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_bag_first">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>150</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_bag_find">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>60</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>найти</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_bag_first">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>181</y>
      <width>461</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_bag_save">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>сохранить</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_bag_second">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>400</y>
      <width>461</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_bag_united">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>610</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_comp_first">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>181</y>
      <width>461</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_comp_second">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>400</y>
      <width>461</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_comp_united">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>640</y>
      <width>461</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1250</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
